libs/deserializer/src/main/java/de/cyface/deserializer/DeserializerFactory.java

```diff
@@ -88,10 +91,11 @@ public static ZippedPhoneDataDeserializer create(final UUID userId, final Path m
      * @param accelerationFiles The files containing the accelerations from the accelerometer for each measurement
      * @param rotationFiles The files containing the rotations from the gyroscope for each measurement
      * @param directionFiles The files containing the directions from the compass for each measurement
+     * @param uploadDate The upload date when the `Measurement`s were uploaded to the collector.
      * @return A {@link UnzippedPhoneDataDeserializer} to read measurements from an unzipped phone export
      */
     public static UnzippedPhoneDataDeserializer create(final UUID userId, final Path measuresDatabase,
-                                                       final List<Path> accelerationFiles, final List<Path> rotationFiles, final List<Path> directionFiles) {
-        return new UnzippedPhoneDataDeserializer(userId, measuresDatabase, accelerationFiles, rotationFiles, directionFiles);
+                                                       final List<Path> accelerationFiles, final List<Path> rotationFiles, final List<Path> directionFiles, final Date uploadDate) {
```

Comment:
Each Parameter on a new line.

Reply:
The later Kotlin auto-conversion would have done this for us, but I'll fix it anyways here manually.